packages/core/src/2d/sprite/SpriteMaterial.ts

```diff
@@ -1,42 +1,41 @@
 "use strict";
 
-import { Shader } from "@oasis-engine/core";
+import { Shader } from "../../shader";
 
 const spriteVertShader = `
 precision highp float;
 
-uniform mat4 matProjection;
-uniform mat4 matView;
+uniform mat4 u_VPMat;
 
-attribute vec3 a_pos;
-attribute vec2 a_uv;
-attribute vec4 a_color;
+attribute vec3 POSITION;
+attribute vec2 TEXCOORD_0;
+attribute vec4 COLOR_0;
 
 varying vec2 v_uv;
 varying vec4 v_color;
 
 void main()
 {
-  gl_Position = matProjection * matView * vec4(a_pos,1.0);
-  v_uv = a_uv;
-  v_color = a_color;
+  gl_Position = u_VPMat * vec4(POSITION, 1.0);
+  v_uv = TEXCOORD_0;
+  v_color = COLOR_0;
 }
 `;
 
 const spriteFragmentShader = `
 precision mediump float;
 precision mediump int;
 
-uniform sampler2D s_diffuse;
+uniform sampler2D u_texture;
 varying vec2 v_uv;
 varying vec4 v_color;
 
 void main()
 {
   // Only use the Alpha of the texture as a mask, so that the tint color can still be controlled to fade out.
-  vec4 baseColor = texture2D(s_diffuse, v_uv);
+  vec4 baseColor = texture2D(u_texture, v_uv);
   gl_FragColor = baseColor * v_color;
 }
 `;
 
-Shader.create("Sprite", spriteVertShader, spriteFragmentShader);
+Shader.create("Sprite-Default", spriteVertShader, spriteFragmentShader);
```

Comment:
The shader name "Sprite" is better than "Sprite-Default"

Reply:
> The shader name "Sprite" is better than "Sprite-Default"

done